cms/static/js/views/metadata.js

```diff
@@ -33,12 +33,17 @@ function(BaseView, _, MetadataModel, AbstractEditor, FileUpload, UploadDialog, V
                             'Float': 'Number',
                             'Integer': 'Number'
                         },
-                        type = model.getType();
+                        type = model.getType(),
+                        options = model.get("options");
```

Comment:
Oh, I see.

This is one reason I prefer to use a separate `var` for each declaration. Also, since this is a forked feature, we should be mindful of minimizing the diff from upstream. We'll get the best of both worlds by changing this to:

```javascript
    // leave type = line alone
var options = model.get('options');
```